### PR TITLE
780 removing skeleton grid swap to flex

### PR DESCRIPTION
**Pull request by dzhibas**
the ones left to fix:

```
$ rg "(columns|container|three|four)" -l

components/what/cli/production.hbs
components/what/cli/example.hbs
components/what/embedded/testimonials.hbs
components/what/embedded/showcase.hbs
components/what/wasm/pitch.hbs
components/what/wasm/js.hbs
components/what/wasm/production.hbs
components/what/wasm/get-started.hbs
```

### CODE REVIEW

File: templates/production/users.hbs [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
-          <div class="flex flex-column justify-between ba br3 b--black mw-100 w-100-l pa3 ma4 max-width-quarter-l">
+          <div class="flex flex-column justify-start ba br3 b--black mw-100 w-100-l pa4 ma4 max-width-quarter-l">
             <div class="user-logo">
               <a href="{{ u.url }}">
                 <img src="/static/images/user-logos/{{ u.logo }}" alt="{{ u.logo }}" />
               </a>
             </div>
             <h4 class="f2 fw6">{{ u.name }}</h4>
-            <p>{{{ u.how }}}</p>
+            <p class="">{{{ u.how }}}</p>
```
Comment (dzhibas) [inline templates/production/users.hbs]: need to remove empty class. was fixing code blocks and highlighting but eventually left it for later

**Comment by dzhibas** [comment]
will pull latest master and fix conflict later

**Comment by skade** [comment]
Please rebase on current master, we just merged the i18n basics. :heart:

**Comment by dzhibas** [comment]
@skade should be good now, checked again after conflict resolve looks o.k.

**Comment by skade** [comment]
:tada: LGTM, merging

**Pull request merged**